pkg/image/source_test.go

```diff
@@ -10,50 +10,56 @@ import (
 
 	"github.com/mitchellh/go-homedir"
 	"github.com/spf13/afero"
+	"github.com/sylabs/sif/v2/pkg/sif"
 )
 
 func TestDetectSource(t *testing.T) {
 	cases := []struct {
 		name             string
+		getFS            func(*testing.T) afero.Fs
 		input            string
 		source           Source
 		expectedLocation string
-		tarPath          string
-		tarPaths         []string
 	}{
 		{
 			name:             "podman-engine",
+			getFS:            getDummyEmpty(),
 			input:            "podman:something:latest",
 			source:           PodmanDaemonSource,
 			expectedLocation: "something:latest",
 		},
 		{
 			name:             "docker-archive",
+			getFS:            getDummyEmpty(),
 			input:            "docker-archive:a/place.tar",
 			source:           DockerTarballSource,
 			expectedLocation: "a/place.tar",
 		},
 		{
 			name:             "docker-engine-by-possible-id",
+			getFS:            getDummyEmpty(),
 			input:            "a5e",
 			source:           UnknownSource,
 			expectedLocation: "",
 		},
 		{
-			name: "docker-engine-impossible-id",
+			name:  "docker-engine-impossible-id",
+			getFS: getDummyEmpty(),
 			// not a valid ID
 			input:            "a5E",
 			source:           UnknownSource,
 			expectedLocation: "",
 		},
 		{
 			name:             "docker-engine",
+			getFS:            getDummyEmpty(),
```

Comment:
maybe just make this the default instead of needing to add it to every test case?